src/neuroglancer/widget/icon.ts

```diff
@@ -24,6 +24,19 @@ export interface MakeIconOptions {
   href?: string;
 }
 
+export interface MakeHoverIconOptions extends MakeIconOptions {
+  svgHover?: string
+}
+
+export function makeHoverIcon(options: MakeHoverIconOptions): HTMLElement {
```

Comment:
this class may be overkill, I wanted the star icon to change to a x to show it was a destructive action. I could replace it with two makeIcons.